samples/go/csv/read_test.go

```diff
@@ -148,6 +148,9 @@ func TestEscapeStructFieldFromCSV(t *testing.T) {
 		"https://picasaweb.google.com/data", "httpspicasawebgooglecomdata",
 		"💩", "",
 		"11 1💩", "",
+		"-- A B", "aB",
+		"-- A --", "A",
```

Comment:
the thought here is that the dashes are bad, so the algorithm would strip the bad characters and apply camel casing as a best effort. so if we strip the dashes we get "A B" then transforming it to camel case it becomes aB.